CancelBooking UseCase/CancelBooking.java

```diff
@@ -146,6 +149,7 @@ public void showCancellationPolicy() {
 }
 // Presentation Layer
 // Main class to handle user interaction for making and canceling reservations
+
```

Comment:
```suggestion
instead of dealing with logic and UI together in the "main" method, it is better to separate concerns into distinct methods and keep "main" only for managing control flow. Also you can add input validation for room number and CPR input to ensure they are not empty or malformed.
```